spec/features/index_page_spec.rb

```diff
@@ -46,4 +47,11 @@
       ]
     )
   end
+
+  it 'takes you to single content item if you click on title of item' do
+    content_data_api_has_metric(base_path: 'path/1', from: (Time.zone.today - 30.days), to: Time.zone.today, metrics: metrics)
```

Comment:
In theory, the `from` and `to` should be the same as the ones specifies in the index page. Are we testing it?